ui/src/dashboards/components/TemplateVariableRow.js

```diff
@@ -30,7 +32,9 @@ const RowValues = ({
   onStartEdit,
   autoFocusTarget,
 }) => {
-  const _values = values.map(({value}) => value).join(', ')
+  const _values = uniq(values.map(({value}) => value))
```

Comment:
Only other thing is that we could in theory use ES6's `Set` instead of pulling in lodash. But it might be less terse overall.